canonical/ubuntu.com
# Wd 17296 dev create solutions open source security cyber resilience act

## Done

- Created Page on CRA

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Visit [ /solutions/open-source-security/cyber-resilience-act](https://ubuntu-com-14560.demos.haus/solutions/open-source-security/cyber-resilience-act)
- Compare with https://www.figma.com/design/8JrAyKnWMU9FPJE0zJT2gu/Canonical.com%2Fsolutions%2Fopen-source-security?node-id=322-3837&node-type=frame&t=s7VX6L8CVxaBok0z-0
- Also correlate with https://docs.google.com/document/d/13VaiP6GGkGB4mE01bIqJ3cg8MfQXulNs9m3cmYb20HQ/edit

## Issue / Card

Fixes [#WD-17296](https://warthogs.atlassian.net/browse/WD-17296)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
